assets-and-images.md

```diff
@@ -248,10 +248,14 @@ Flutter also uses native platform mechanisms to draw transitional launch screens
 
 To add a "splash screen" to your Flutter application, navigate to `.../android/app/src/main`. In `res/drawable/launch_background.xml`, You can use this [layer list drawable](https://developer.android.com/guide/topics/resources/drawable-resource.html#LayerList) XML to customize the look of your launch screen. The existing template provides an example for adding a image to to the middle of a white splash screen in commented code. You can uncomment it or use other [drawables](https://developer.android.com/guide/topics/resources/drawable-resource.html) to achieve the intended effect.
 
+You can disable the effect of showing the splash screen until Flutter renders the first frame by setting the default first activity's meta-data `io.flutter.app.android.SplashScreenUntilFirstFrame` value to false in the `AndroidManifest.xml` or by not setting a `android:windowBackground` altogether on the activity via the default `@style/LaunchTheme` theme.
```

Comment:
You can delay showing the splash screen until right after Flutter renders the first frame of your app. <does this help with apps that are already fast to launch? does it help remove the default blank screen?>. To delay the splash screen, modify the default first activity entry in AndroidManifest.xml in one of the following ways:
* Set the value of `io.flutter.app.android.SplashScreenUntilFirstFrame` to `false`
* If the `android:theme` attribute exists, do not set `android:windowBackground` for the default `@style/LaunchTheme` theme